invenio_accounts/api.py

```diff
@@ -8,25 +8,60 @@
 
 """API objects for Invenio Accounts."""
 
-from collections import defaultdict
+
+class Session:
+    """Session object for DB Users change history."""
+
+    def __init__(self):
+        self.updated_users = []
+        self.updated_roles = []
+        self.deleted_users = []
+        self.deleted_roles = []
+        self.indexed = False
```

Comment:
are these 2 properties used elsewhere?

Reply:
Yes, in [`invenio-users-resources`](https://github.com/inveniosoftware/invenio-users-resources/pull/75/files#diff-2dbb78424a207aae3daba71b31a89e424293633e65090f2239f68dcfe5a756d9R66) custom signal handler to take care of the indexing and in [`invenio-communities`](https://github.com/inveniosoftware/invenio-communities/pull/945/files#diff-6c07ffce9d2dd97c912da89fd4869787bdb49626a3fa278d466117cc8e489941R130) to invalidate the cache.